content/en/docs/tasks/security/cert-management/custom-ca-k8s/index.md

```diff
@@ -157,6 +157,7 @@ Refer to the [Kubernetes CSR documentation](https://kubernetes.io/docs/reference
       data:
       root-cert.pem: <tls.cert from the step above>
       EOF
+    $ kubectl create namespace istio-system
```

Comment:
Everything else on the page seems to assume istio-system has already been created.  Is there a reason this step needs to have it added?  I assume it's because the cert must be there before you run `istioctl install` ?

Reply:
removed this command, thanks @craigbox